lib/packages.ts

```diff
@@ -152,14 +152,15 @@ const packageJsonPaths = [
   ..._findPrimaryPackageJsonFiles(path.join(__dirname, '..', 'packages'), excludeRe),
 ];
 
-function _exec(cmd: string) {
-  return execSync(cmd).toString().trim();
+function _exec(cmd: string, opts?: { cwd?: string }) {
+  return execSync(cmd, opts).toString().trim();
 }
 
 let gitShaCache: string;
 function _getSnapshotHash(_pkg: PackageInfo): string {
   if (!gitShaCache) {
-    gitShaCache = _exec('git log --format=%h -n1');
+    const opts = { cwd: __dirname }; // Ensure we call git from within this repo
+    gitShaCache = _exec('git log --format=%h -n1', opts);
```

Comment:
In the snapshot script, there's a `process.chdir()` outside of the git repo, which causes this calculation of the hash to fail. This just makes sure it gets run inside of the git repo. This didn't occur previously because we loaded the legacy build script cached the sha value before that chdir occurred.